skimage/measure/_regionprops.py

```diff
@@ -358,22 +358,73 @@ def __eq__(self, other):
         return True
 
 
+def to_dict(regions, want=['label', 'bbox'], separator='-', cancel_auto=False):
+    """Convert image regions properties into a dictionary
+
+    Parameters
+    ----------
+    regions : (N,) list
+        _RegionProperties
+    want : list, optional
+        str of a property the user wants to include
+    separator : str, optional
+        used in joining coordinates
+    cancel_auto : bool, optional
+        prevents autmatic adding of label and bbox
+
+
+    Output
+    ------
+    A dictionary with properties in accordance to the wanted properties
```

Comment:
Add a parameter name, and more detail: "Dictionary mapping property names to an array of values of that property, one per region. This dictionary can be used as input to a pandas DataFrame, with keys mapping to columns."